### ISSUE TITLE
Center text

### ISSUE THREAD

**Issue by reni99** [jsdf/react-native-htmlview#58]
Hi everyone,

Is it possible to center text? If I have something like this:
`<p> this is some text... bla bla bla </p>`

How would I need to set the style to set the text alignment to center? 

I tried something like this, which did not work:

```
let text = "<p>this is some text... bla bla bla </p>";

<View style={styles.wrapper}>
    <HTMLView value={text} stylesheet={styles}/>
</View>

const styles = StyleSheet.create({
    wrapper: {
        alignItems: "center",
  },
   p: {
     textAlign: "center",
}
);
```

I also tried to set the alignment directly with `<p align="right">`. Did not work either...

Any help would be appreciated :)

**Comment by nobalmohan** [comment]
```
const styles = StyleSheet.create({
    wrapper: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center'
  }
});
```

**Comment by dan-katz** [comment]
Is there any progress on this? textAlign: 'center' still doesn't work for me.

**Comment by shaibar** [comment]
@reni99 
I just used the following:
### The HTML view usage using the renderNode prop function for custom rendering
```
<HTMLView value={this.props.item.description()} stylesheet={descriptionStyles}
                                          renderNode={this.renderDescription}/>
```

### Render my node function
```
renderDescription = (node, index, list, parent) => {
        if (node.type === 'text') {
            return (
                <Text key={index} style={descriptionStyles.p}>
                    {entities.decodeHTML(node.data)}
                </Text>);
        } else if (node.type === 'tag') {
            return (
                <View>
                    {node.children.map(this.renderDescription)}
                </View>)
        }
        return false;
    }
```

### Style
```
const descriptionStyles = StyleSheet.create({
    p: {
        color: 'white',
        fontSize: 18,
        textAlign: 'center',
    },
});
```

**Comment by hchen1202** [comment]
Has anyone found the solution?

**Comment by indrasantosa** [comment]
Here is what I did.

Define the element like this
```jsx
const htmlContent = `<a href="http://path-to-your-link.com">link</a>`;
```

And use it like this
```jsx
<View style={{ flex: 1, flexDirection: 'row', alignItems: 'center', justifyContent: 'center' }}>
   <HTMLView
      value={htmlContent}
   />
</View>
```

**Comment by flavordaaave** [comment]
The `HTMLView` accepts an (undocumented) `style` prop. So you could also apply the wrapper style directly to the `HTMLView`:
```
<HTMLView
  style={{ flex: 1, flexDirection: 'row', alignItems: 'center', justifyContent: 'center' }}
  value={ htmlString } />
```

**Comment by xsgkaka** [comment]
i got the solution
i use the 0.9.0. change the line 96 in htmlToElement .to like this
`<View key={index} onPress={linkPressHandler}>
            {linebreakBefore}
            {listItemPrefix}
            {domToElement(node.children, node)}
            {linebreakAfter}
          </View>`

**Comment by RobPando** [comment]
@xsgkaka This throws an error of `RawText "" must be wrapped in an explicit <Text> component` and @flavordaaave solution only center aligns the first tag if you have multiple tags. I found a global solution, I created a PR for it #116 this makes the entire value of the component align text center. due to the implementation it is complicated to make them independently align text. But if you are looking to align the entire value then this works good.
